Makefile

```diff
@@ -317,18 +317,42 @@ else
 	$(DOCKER_HELM_CMD) $(LINT_CHART_CMD)
 endif
 
-PUBLISH_CHART_CMD := bash -c ' \
+PUBLISH_RC_CHART_CMD := bash -c ' \
 	cd contrib/k8s/charts \
 	&& rm -rf repo \
 	&& mkdir repo \
 	&& cd repo \
-	&& sed -i s/0.0.1/$(REL_VERSION)/g ../open-service-broker-azure/Chart.yaml \
+	&& sed -i "s/^version:.*/version: 0.0.1+$(GIT_VERSION)/g" ../open-service-broker-azure/Chart.yaml \
```

Comment:
Helm requires versions to be a legitimate semver string, so we'll assign special meaning to 0.0.1, because we can't get away with using a value like "rc" or "canary". (I tried.)